go/private/go_toolchain.bzl

```diff
@@ -92,7 +92,7 @@ go_toolchain = rule(
     provides = [platform_common.ToolchainInfo],
 )
 
-def declare_toolchains(host, sdk, builder):
+def declare_toolchains(host, sdk, builder, sdk_version_settings):
```

Comment:
I would prefer singular here as we expect a single label literal (even though it will usually be a group).